src/tds/tls.c

```diff
@@ -18,11 +18,11 @@
  * Boston, MA 02111-1307, USA.
  */
 
-#include <config.h>
-
```

Comment:
Why? This file is not supposed to include files so the `_WIN32_WINNT` definition won't change the result.

Reply:
Ah, right, this warning's specific to my setup -- in the course of integrating FreeTDS, I supply a `config.h` that sits on top of NCBI C++ Toolkit's configuration headers and as such indirectly pulls in a conditional `_WIN32_WINNT` definition.